forms/new-chapter.md

```diff
@@ -27,10 +27,11 @@ Please fill in the following form to the best of your knowledge, and keep in min
 
 ## Programming
 
-- `When do you plan to have your first event?`
 - `What type of event(s) do you plan to run?`
+- `How many events are you planning to do per year?`
+- `When do plan to have your first event`?
 - `Do you have any specific support requests? For example, does your community speak a language other than English and will you want translation support?`
-- `Do you need mentorship from chapters in cities near by? If so which chapters?`
+- `Have you received mentorship from existing chapter organizers? If not, please join us in Slack under #prospective-organizers channel`
```

Comment:
I rephrased this question. I think they should have seek mentorship before filling in this form? What do you think?